fbw-common/src/typings/fs-base-ui/html_ui/JS/common.d.ts

```diff
@@ -868,8 +868,8 @@ declare global {
     let g_AtlasMgr: AltasElementsMgr;
 
     namespace InputBar {
-        const MENU_BUTTON_A: 'MENU_VALID';
-        const MENU_BUTTON_B: 'MENU_BACK';
+        const MENU_BUTTON_A: 'MENU_VALID' | 'KEY_MENU_SR_VALID';
+        const MENU_BUTTON_B: 'MENU_BACK' | 'KEY_MENU_SR_BACK';
```

Comment:
Label which is MSFS2020/MSFS2024 with a comment

Reply:
To me this was clear… I guess I can add an explicit comment.